src/main/java/uk/gov/nationalarchives/pdi/step/xml/CanonicalizationResult.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.nationalarchives.pdi.step.xml;
+
+public class CanonicalizationResult {
```

Comment:
I am just wondering as to the purpose of the error count?
If it is more tracking multiple errors, why not have a `List<String> errorMessages` instead, and then just concatenate them when required for serialization?

Reply:
Actually there can only ever be one error. The canonicalization either works or throws an exception. I have removed the error count.